src/deployment-guides/tap.md

```diff
@@ -0,0 +1,799 @@
+
+# tap available package list
+tanzu package available list --namespace "${TAP_NAMESPACE}"
+```
+
+### <a id=tap-profile-build> </a>Step 3: Install Tanzu Application Platform build profile
+
+Provide following user inputs to set environments variables into commands and execute them to install build profile
+
+* `TAP_REGISTRY_SERVER` - uri of registry server
+* `TAP_REGISTRY_USER` - registry user
+* `TAP_REGISTRY_PASSWORD` - registry password
+
+```bash
+set -e
+
+export TAP_NAMESPACE="tap-install"
+export TAP_REGISTRY_SERVER=<registry server uri>
+export TAP_REGISTRY_USER=<registry user>
+export TAP_REGISTRY_PASSWORD=<registry password>
+
+cat <<EOF | tee tap-values-build.yaml
+profile: full
+ceip_policy_disclosed: true
+buildservice:
+  kp_default_repository: "${TAP_REGISTRY_SERVER}/build-service"
+  kp_default_repository_username: "${TAP_REGISTRY_USER}"
+  kp_default_repository_password: "${TAP_REGISTRY_PASSWORD}"
+  tanzunet_username: "${TANZU_NET_USER}"
+  tanzunet_password: "${TANZU_NET_PASSWORD}"
+supply_chain: basic
+ootb_supply_chain_basic:
+  registry:
+    server: "${TAP_REGISTRY_SERVER}"
+    repository: "supply-chain"
+  gitops:
+    ssh_secret: ""
+  cluster_builder: default
+  service_account: default
+tap_gui:
+  service_type: LoadBalancer
+  
+metadata_store:
+  app_service_type: LoadBalancer
+
+excluded_packages:
+  - accelerator.apps.tanzu.vmware.com
+  - run.appliveview.tanzu.vmware.com
+  - api-portal.tanzu.vmware.com
+  - cnrs.tanzu.vmware.com
+  - ootb-delivery-basic.tanzu.vmware.com
+  - developer-conventions.tanzu.vmware.com
+  - image-policy-webhook.signing.apps.tanzu.vmware.com
+  - learningcenter.tanzu.vmware.com
+  - workshops.learningcenter.tanzu.vmware.com
+  - services-toolkit.tanzu.vmware.com
+  - service-bindings.labs.vmware.com
+  - tap-gui.tanzu.vmware.com
+
+EOF
+
+tanzu package install tap -p tap.tanzu.vmware.com -v 1.0.0 --values-file tap-values-build.yaml -n "${TAP_NAMESPACE}"
+tanzu package installed get tap -n "${TAP_NAMESPACE}"
+
+# check all build cluster package installed succesfully
+tanzu package installed list -A
+```
+
+### <a id=tap-dev-namespace> </a>Step 4: Configure developer namespaces to use installed packages
+
+Execute the following commands to configure a developer namespace:
+
+```bash
+set -e
+
+export TAP_DEV_NAMESPACE="default"
+export TAP_REGISTRY_SERVER=<registry server uri>
+export TAP_REGISTRY_USER=<registry user>
+export TAP_REGISTRY_PASSWORD=<registry password>
+
+tanzu secret registry add registry-credentials --server "${TAP_REGISTRY_SERVER}" --username "${TAP_REGISTRY_USER}" --password "${TAP_REGISTRY_PASSWORD}" --namespace "${TAP_DEV_NAMESPACE}"
+
+cat <<EOF | kubectl -n "${TAP_DEV_NAMESPACE}" apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: tap-registry
+  annotations:
+    secretgen.carvel.dev/image-pull-secret: ""
+type: kubernetes.io/dockerconfigjson
+data:
+  .dockerconfigjson: e30K
```

Comment:
This does not look like a valid `.dockerconfigjson`. What is going on here?

Reply:
@craigtracey - check this link of tap for dockerconfigjson config, here you can see the same steps i have provided into tap guide.  https://docs.vmware.com/en/Tanzu-Application-Platform/1.0/tap/GUID-install-components.html?hWord=N4IghgNiBcICYHsDGBrApgJyQgdgMwEsBzAKwGdcQBfIA  .  See this section "Set up developer namespaces to use installed packages"